src/routes/docs/quick-starts/python/+page.markdoc

```diff
@@ -201,22 +201,58 @@ def seed_database():
 {% /section %}
 {% section #step-7 step=7 title="Retrieve documents" %}
 
-Create a function to retrieve the mock todo data, 
+Create a function to retrieve the mock todo data,
 then execute the functions in `_main_`.
 
 ```py
+from appwrite.query import Query
+
 def get_todos():
+  # Retrieve documents (default limit is 25)
+  todos = databases.list_documents(
+    database_id=todoDatabase['$id'],
+    collection_id=todoCollection['$id'],
+    queries=[]
+  )
+  print("Todos:")
+  for todo in todos['documents']:
+    print(f"Title: {todo['title']}\nDescription: {todo['description']}\nIs Todo Complete: {todo['isComplete']}\n\n")
+
+def get_completed_todos():
+  # Use queries to filter completed todos with pagination
   todos = databases.list_documents(
     database_id=todoDatabase['$id'],
-    collection_id=todoCollection['$id']
+    collection_id=todoCollection['$id'],
```

Comment:
can you also fix `todoDatabase` and `todoCollection` to be snake case to match python conventions?